src/webviews/main.ts

```diff
@@ -105,11 +104,12 @@ export async function createVueWebview<TRequest, TResponse>(params: WebviewParam
 </html>`
 
     if (params.initialCalls) {
-        // TODO: workaround for cloud9 webviews, remove after cloud9 issue is fixed.
-        if (isCloud9()) {
-            await new Promise(resolve => setTimeout(resolve, 500))
-        }
-        for (const call of params.initialCalls) view.webview.postMessage(call)
+        const d = view.webview.onDidReceiveMessage((message: any) => {
+            if (message.command === 'initialized') {
+                for (const call of params.initialCalls!) view.webview.postMessage(call)
+                d.dispose()
```

Comment:
Unsure if we should dispose this, specifically thinking of what happens if we navigate to and away from the webview. Maybe tie it to `view.onDidDispose`?

Reply:
Presumably these callbacks were only meant to be fired once, which is why it's being disposed.

Reply:
Since the entire view is re-drawn when moving away from the webview, the dispose should be moved to the webview like you said.